plugin/core/logging_notify.py

```diff
@@ -0,0 +1,20 @@
+from __future__ import annotations
+import sublime
+from .settings import userprefs
+
+
+def notify_error(window: sublime.Window | None, message: str, status_message: str = 'LSP: see console log…',
+                 severe: bool = True) -> None:
```

Comment:
You never used that argument in the function calls, so I think we don't need that and it can be removed. The difference between `sublime.error_message` and `sublime.message_dialog` is just an additional icon anyways (at least on Windows), so it's probably not very important.

Reply:
k, removed

I've never used it because you said that all current messages are the same, so they all got bumped to "severe" in the process of converting to a single function.

I haven't reviewed all the messages to see whether a different type is justified. This again was more future-proof in case some messages will be deemed less severe

Reply:
Yeah but if we need it in the future, we can simply add it then. I'm not gonna say that I have never done this myself, but in general it's probably not very useful to add unused code with the reasoning that it could potentionally be used in the future.